cloud.foundry.cli/src/main/java/cloud/foundry/cli/crosscutting/mapping/CfOperationsCreator.java

```diff
@@ -11,7 +11,7 @@
 public class CfOperationsCreator {
 
     public static DefaultCloudFoundryOperations createCfOperations(
```

Comment:
In this class docstrings are still missing everywhere. Just a general note.

Reply:
The class remained unchanged on this branch. I would suggest that we add the missing documentation in the next refactoring process.

Reply:
Sounds good to me. Marking as resolved.